### PR TITLE
Update SGCraft.zs

### PR DESCRIPTION
**Pull request by GTNH-Colen**

### CODE REVIEW

**Comment by leagris** [comment]
Any description of this patch?

**Comment by boubou19** [comment]
pushing the assline where it's supposed to belong: endgame

**Comment by Dream-Master** [comment]
Not set computation to 1. it was 32 before

**Comment by GTNH-Afx237v7** [comment]
Make Stargate difficult again!

**Review by Dream-Master — APPROVED** [review]

**Comment by Sphyix** [comment]
> Any description of this patch?

Yes, "Shitty patch number 15"
Another nerf just for the sake of it, adding nothing to the gameplay other than more grind.

Stargate was moved to UEV, a completely empty tier that literally exists just for energy hatches, since there's 0 content there.
Instead of trying to add content, a casual nerf making the game more grindy and more time consuming was done, apparently that's how content is made, just nerf everything to oblivion.

Even worse, since ppl were obviously going to complain, this PR was completely nuked from discord.

So I'm here complaining. Recipe should use UHV components instead of UEV, since UEV is an empty tier.
At least there's a bit of content to UHV, so that's the real endgame now.
Making it UEV just makes it so grindy it's not even funny...

**Comment by Sphyix** [comment]
I propose to revert the change and add circuits like biowwares starting from UV and UHV before nerfing this, like, actually adding content for endgame.
This change just makes no sense as it is right now.

**Comment by GTNH-Colen** [comment]
Firstly changes were agreed to by Dream in DMs, boubou was also aware as I discussed it in vc with him. Secondly the VMs have made stargates significantly easier to craft and we can see the direct impact of that given no stargate was crafted in the 5 years preceding the first void miners. Then suddenly within a year or so we have 6ish people who have a stargate. It is a meme item, if this was balance surrounding a normal useful item like the draconic magnet I'd agree.

In short this is a vanity project that due to additional content has become easier, as such it was pushed back to be where it was meant to be: meme status.

**Comment by GTNH-Colen** [comment]
Also side note: still in discord.

Screenshot 2022-01-19 at 23 19 14
